src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,154 @@
+import PropTypes from 'prop-types'
+import Link from 'gatsby-link'
+
+function Pagination({ pageContext, type }) {
+  let { currentPage, totalPages, minPage, maxPage, path } = pageContext
+
+  const apiPageChange = value => {
+    if (value !== currentPage) pageContext.formData(pageContext.formResponse, value)
+  }
+
+  const direction = value => {
+    let count = value.target.innerText === 'More' ? 10 : -10
+    pageContext.setPageContext({
+      currentPage: currentPage,
+      totalPages: totalPages,
+      minPage: minPage + count,
+      maxPage:
+        count === 'More'
+          ? maxPage + 10 > totalPages
+            ? totalPages
+            : maxPage + 10
+          : maxPage + count,
+      setPageContext: pageContext.setPageContext,
+      formResponse: pageContext.formResponse,
+      formData: pageContext.formData,
+    })
+  }
+
+  if (type === 'route') {
```

Comment:
Please remove one of these if blocks in favor of returning one `type` from the component by default.

Reply:
Should we have two pagination components. One for api and one for route?

Reply:
My bad, poor wording, I would like to see one if statement to be removed and `if(type==='route')` return that early and always return the other type.

Reply:
understood 😅

Reply:
done